src/lib.rs

```diff
@@ -250,28 +250,13 @@ impl<T> Slab<T> {
     /// # use slab::*;
     /// let slab: Slab<i32> = Slab::new();
     /// ```
-    #[cfg(not(slab_no_const_vec_new))]
     pub const fn new() -> Self {
         Self {
             entries: Vec::new(),
             next: 0,
             len: 0,
         }
     }
-    /// Construct a new, empty `Slab`.
-    ///
-    /// The function does not allocate and the returned slab will have no
-    /// capacity until `insert` is called or capacity is explicitly reserved.
-    ///
-    /// This is `const fn` on Rust 1.39+.
```

Comment:
There is the same wording in `const fn new` as this one, could you remove it?

Reply:
Done